docs/source/broadcasting.md

```diff
@@ -117,7 +117,7 @@ Verifier server listening in port 5000
 If you are running the verifier on the same machine as the broadcaster, then you can run the below command in order to connect the broadcaster with the verifier:
 
 ```
-livepeer -network rinkeby -broadcaster -verifierUrl http://localhost:5000/verify -verifierPath ~/verification-classifier/stream
+livepeer -network rinkeby -ethUrl <ETH_RPC_URL> -broadcaster -verifierUrl http://localhost:5000/verify -verifierPath ~/verification-classifier/stream
```

Comment:
Maybe a brief note linking to the quickstart section describing how to obtain an ethUrl?

Reply:
Added sections at the beginning of both "Transcoding" and "Broadcasting" linking to the quick start section describing how to obtain an ethUrl in e82f4ac